FastSimulation/SimplifiedGeometryPropagator/src/ParticleManager.cc

```diff
@@ -91,7 +91,10 @@ std::unique_ptr<fastsim::Particle> fastsim::ParticleManager::nextParticle(const
         const HepPDT::ParticleData * particleData = particleDataTable_->particle(HepPDT::ParticleID(particle->pdgId()));
         if(!particleData)
         {
-            throw cms::Exception("fastsim::ParticleManager") << "unknown pdg id: " << particle->pdgId() << std::endl;
+            // in very few events the Decayer (pythia) produces high mass resonances that are for some reason not present in the table (even though they should technically be)
+            // they have short lifetimes, so decay them right away (charge and lifetime cannot be taken from table)
+            particle->setRemainingProperLifeTimeC(0.);
+            particle->setCharge(0.);
```

Comment:
@skurz @ssekmen are these sufficient protections to get rid of the issue? Which particles are causing this in a standard model run, and why are there?

Reply:
So far we've observed particles with pdgIDs 9010221 and 30443. They can be produced if a non-stable particle is decayed https://github.com/skurz/cmssw/blob/157e9a58bcfcc99d62a9744e34edec23d5f0b2ab/FastSimulation/SimplifiedGeometryPropagator/src/Decayer.cc#L33

Reply:
For an initial test of 6000 events, there was no crash with the fix (whereas the current version crashes).